t/redirects.rb

```diff
@@ -0,0 +1,10 @@
+require 'test_helper'
+
+describe 'redirecting links' do
+  around { |test| VCR.use_cassette('liff', &test) }
+  subject { WikiData.ids_from_pages('en', ['The Meaning of Liff', 'The Deeper Meaning of Liff']) }
```

Comment:
Multiple redirects to a common article are handled in the first commit (3f7cfe3)

The part that swaps the `from` and `to`:
``redirected_from = (response.data['redirects'] || []).map { |h| [h['from'], h['to']] }``

The change is not separate to the change the commit message describes.

I should make the change separately but I'm not sure what the best thing to do would be: should I split the commit and keep both changes in this PR or should I revert the change in a fix up and make it in a separate PR that targets this branch?